backend/src/service/Jimmi.class.ts

```diff
@@ -0,0 +1,9 @@
+class Jimmi {
+  public readonly id: string;
+
+  constructor() {
+    this.id = globalThis.crypto.randomUUID();
```

Comment:
```suggestion
    this.id = crypto.randomUUID();
```
Do we need explicit `globalThis`?